### PR TITLE
Update tube rack scan upload page guidance text

### PR DESCRIPTION
**Pull request by KatyTaylor**
- in cardinal cell banking pipeline
- it was inaccurate

### CODE REVIEW

File: app/views/tube_creation/pooled_tubes_by_sample.html.erb [hunk 1]
```diff
@@ -21,12 +15,12 @@
...
+          <tr><td>D2</td><td>NO READ</td></tr>
+          <tr><td>E2</td><td>AB00000013</td></tr>
           <tr><td>etc...</td></tr>
         </tbody>
       </table>
-      <p>NB. Columns headers and all 96 positions in the rack must be represented in the file. If a tube is not present in a position, the system expects to see a NoScan value in the barcode column.</p>
+      <p>NB. The system will also understand the value NO READ instead of a barcode, if the scan was not successful in that position, and ignore that tube when deciding where to transfer material.</p>
     <% end %>
   <% end %>
   <%= sidebar do %>
```
Comment (KatyTaylor) [inline app/views/tube_creation/pooled_tubes_by_sample.html.erb:26]: New:
Screenshot 2021-12-20 at 09 57 46

Comment (harrietc52) [reply app/views/tube_creation/pooled_tubes_by_sample.html.erb:26]: Very useful

File: app/views/tube_creation/pooled_tubes_by_sample.html.erb [hunk 2]
```diff
@@ -21,12 +15,12 @@
...
           <tr><td>C2</td><td>AB00000011</td></tr>
-          <tr><td>D2</td><td>NoScan</td></tr>
-          <tr><td>E2</td><td>NoScan</td></tr>
+          <tr><td>D2</td><td>NO READ</td></tr>
+          <tr><td>E2</td><td>AB00000013</td></tr>
           <tr><td>etc...</td></tr>
         </tbody>
       </table>
-      <p>NB. Columns headers and all 96 positions in the rack must be represented in the file. If a tube is not present in a position, the system expects to see a NoScan value in the barcode column.</p>
+      <p>NB. The system will also understand the value NO READ instead of a barcode, if the scan was not successful in that position, and ignore that tube when deciding where to transfer material.</p>
```
Comment (harrietc52) [inline app/views/tube_creation/pooled_tubes_by_sample.html.erb:23]: Sorry I don't fully understand this sentence, particularly the `, if the scan was not successful in that position,` bit. Might just be me

File: app/views/tube_creation/pooled_tubes_by_sample.html.erb [hunk 3]
```diff
@@ -3,12 +3,6 @@
     <%= card title: 'Help' do %>
       <p>Upload the tube rack scan csv file describing the positions and barcodes of the tubes. An example is shown below:</p>
       <table class='table table-sm'>
-        <thead>
```
Comment (harrietc52) [inline app/views/tube_creation/pooled_tubes_by_sample.html.erb:6]: Are the headers not needed anymore

**Review by harrietc52 — APPROVED** [review]

File: app/views/tube_creation/pooled_tubes_by_sample.html.erb (hunk 3 above)
Comment (KatyTaylor) [reply app/views/tube_creation/pooled_tubes_by_sample.html.erb:6]: Yeah I'm not sure why, but the guidance was wrong - it actually will error if it has headers

File: app/views/tube_creation/pooled_tubes_by_sample.html.erb (hunk 2 above)
Comment (KatyTaylor) [reply app/views/tube_creation/pooled_tubes_by_sample.html.erb:23]: OK thanks, I will try to re-word it

Comment (KatyTaylor) [reply app/views/tube_creation/pooled_tubes_by_sample.html.erb]: Updated - any better?

**Pull request merged**